src/System.Security.Cryptography.Cng/tests/ECDsaCngTests.cs

```diff
@@ -2,98 +2,88 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Collections.Generic;
-using System.Security.Cryptography;
-
+using System.Security.Cryptography.EcDsa.Tests;
 using Test.Cryptography;
 using Xunit;
 
 namespace System.Security.Cryptography.Cng.Tests
 {
-    public static class EcDSACngTests
+    public class ECDsaCngTests : ECDsaTestsBase
     {
         [Fact]
-        public static void TestPositiveVerify256()
+        public void TestPositive256WithBlob()
```

Comment:
Hmm there was some inconsistency already and there still is. The preexisting common\ECDsaTests.cs tests were instance-based which I left that way. And at one point ECDsaTestsBase included tests which requires instance methods for those to run, so I started converting to instance methods, however since then those ECDsaTestsBase common tests were removed and placed into a different common class (import). I will add back the statics here and in a couple other random tests that had instance methods instead of statics (but leave common\ECDsaTests.cs instance-based)